diff --git a/Box2D/Particle/b2ParticleGroup.ts b/Box2D/Particle/b2ParticleGroup.ts
index 6131270133e44b986bf9bfd43c818a784543e451..dd4b0ef9a3cec8551a573ef85f4a8207e96ac009 100644
--- a/Box2D/Particle/b2ParticleGroup.ts
+++ b/Box2D/Particle/b2ParticleGroup.ts
@@ -26,6 +26,7 @@ import { b2ParticleFlag } from "./b2Particle";
 import { b2ParticleSystem } from "./b2ParticleSystem";
 
 export enum b2ParticleGroupFlag {
+  b2_empty = 0,
   /// Prevents overlapping or leaking.
   b2_solidParticleGroup = 1 << 0,
   /// Keeps its shape.
